config.toml

```
@@ -0,0 +1,19 @@
+username = "my-user"
+repository = "https://github.com/my-org/my-repo"
```

Just renaming it for clarity
```suggestion
template = "my-org/my-repo"
```